bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -16,5 +16,16 @@ def handle_log_statement(gast):
         return "echo"
     
     def handle_func_call(gast):
+        # bash has no way to directly print arrays. This logic returns an error for such behavior
+        # TODO: maybe we should run code after translation to make sure it compiles and throw errors if it does NOT
+        if gast["value"]["type"] == "logStatement" and general_helpers.arr_in_list(gast["args"]):
+            return "impossibleTranslationError: direct translation does not exist" # TODO: streamline error message as part of refactor
```

Comment:
Nit: I'd move the #TODO about the return. And also for all these TODO's I would put #TODO(jackdavidweber) or #TODO(taigaTicket#)